### DISCUSSION TITLE
public apps using unpoly ?

### DISCUSSION THREAD

**Discussion by simosentissi** [unpoly/unpoly#232] [General]
Any listing/accolades of **public** applications using unpoly ?

So far the only one that I found from github/twitter: 
- https://agorakit.org/

If it doesn't exist already: It would be good to have such a listing in order to promote and raise confidence (eliminate hesitancy) in this awesome tool/framework.

https://github.com/agorakit/awesome-unpoly

Cheers!

**Comment by triskweline** [comment]
Here are a few more that I know of:

- https://steadyhq.com/
- https://www.volkswagen-newsroom.com/
- https://makandra.com/
- https://makandracards.com/makandra

**Comment by philippejadin** [comment]
As for https://github.com/agorakit/awesome-unpoly this is just a placeholder I wanted to share when there is more info, then propose to transfer it under the official unpoly repository, but so far I did nothing :-). I'll ping this place when there is something worth enhancing.
